### PR TITLE
remove typescript

### PR DESCRIPTION
**Pull request by aleks-and**

### CODE REVIEW

**Review by DeadByElpy — CHANGES_REQUESTED** [review]
please add spaces after "(" and before ")" symbols

File: src/js/components/LogItem/utils/createLogTitle.jsx [hunk 1]
```diff
@@ -61,8 +63,8 @@ const createLogTitle = (log: ILogItem): ReactElement => {
                             <HighlightedText>{log.data.commands.length}</HighlightedText>
                             {'; '}
                             {
-                                hasLrc(log.commandType) &&
-                                log.data.lrc.expected !== log.data.lrc.actual
+                                hasLrc(log.commandType)
```
Comment (DarkPark) [inline src/js/components/LogItem/utils/createLogTitle.jsx:66]: such simple cases it's possible to write in a single line

Comment (aleks-and) [reply src/js/components/LogItem/utils/createLogTitle.jsx]: updated as suggested

**Comment by aleks-and** [comment]
> please add spaces after "(" and before ")" symbols

done

File: eslint-custom-rules/rules/space-in-control-parens.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        type: 'layout',
+        docs: {
+            description: 'Enforce spaces inside parentheses of control statements.',
+            category: 'Stylistic issues',
+            recommended: false
+        },
+        fixable: 'whitespace',
+        schema: []
+    },
+    create ( context ) {
```
Comment (DarkPark) [inline eslint-custom-rules/rules/space-in-control-parens.js:12]:
```suggestion

    create ( context ) {
```

File: eslint-custom-rules/rules/space-in-control-parens.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+                return {
+                    openingParen: target.type !== 'CatchClause'
+                        ? sourceCode.getTokenBefore(target, {filter: token => token.value === '('})
+                        : null,
+                    closingParen: target.type !== 'CatchClause'
+                        ? sourceCode.getTokenAfter(target, {filter: token => token.value === ')'})
+                        : null
+                };
+            };
+
```
Comment (DarkPark) [inline eslint-custom-rules/rules/space-in-control-parens.js:56]:
```suggestion
```

File: eslint-custom-rules/rules/space-in-control-parens.js (hunk 2 above)
Comment (aleks-and) [reply eslint-custom-rules/rules/space-in-control-parens.js:13]: done

File: eslint-custom-rules/rules/space-in-control-parens.js [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+            ForStatement: checkSpacing,
+            WhileStatement: checkSpacing,
+            DoWhileStatement: checkSpacing,
+            ForInStatement: checkSpacing,
+            ForOfStatement: checkSpacing,
+            CatchClause: checkSpacing,
+            SwitchStatement: checkSpacing
+        };
+    }
+
```
[the remark this answers is not in the document]
Comment (aleks-and) [reply eslint-custom-rules/rules/space-in-control-parens.js]: done

File: eslint-custom-rules/rules/space-in-control-parens.js (hunk 3 above)
Comment (aleks-and) [reply eslint-custom-rules/rules/space-in-control-parens.js:57]: done

**Review by DarkPark — APPROVED** [review]

**Review by DeadByElpy — APPROVED** [review]

**Pull request merged**